modules/140-user-authz/templates/cluster-role-bindings.yaml

```diff
@@ -1,28 +1,41 @@
-{{- range $crd := .Values.userAuthz.internal.crds }}
-  {{- if $crd.spec.additionalRoles }}
-    {{- range $additional_role := $crd.spec.additionalRoles }}
+{{- define "rbac.namespace" }}
```

Comment:
If you know how to make this file with templates less cumbersome, you are welcome! :)

Reply:
Tried my best:D